### DISCUSSION TITLE
Add some new example of Vanilla JS/TS

### DISCUSSION THREAD

**Discussion by baboon-king** [TanStack/table#4323] [Ideas]
I not use React/Vue3/more framework 

so I don't know how to start

**Comment by marceloverdijk** [comment]
Did you made progress on this or found some (external) references?
I'm looking into using vanilla TS without a framework and I'm looking for some getting started examples (including filtering and sorting).

**Reply by baboon-king** [reply]
sorry! 

I used [ag-grid](https://github.com/ag-grid/ag-grid) at now
Ag-Grid has "vue 2" adapter 🤩


---
I originally used it in "vue 2". 
But I didn't find the "vue 2" adapter（only Vue3） in the documentation
So I wanted to use the "Vanilla JS/TS" Way

**Comment by marceloverdijk** [comment]
More info here: https://github.com/TanStack/table/discussions/4928
